src/components/Alteration/DocumentsDelivery.vue

```diff
@@ -1,53 +1,53 @@
 <template>
-  <div class="pb-6" id="document-delivery-section">
+  <section id="document-delivery-section">
     <h2>{{sectionNumber}} Documents Delivery</h2>
-    <div class="document-info pt-4 pb-4">Copies of the {{ getFilingName.toLowerCase() }} documents will be sent
-      to the following email address listed below.</div>
+
+    <div class="document-info py-4">
+      Copies of the {{ getFilingName.toLowerCase() }} documents will be sent
+      to the following email address listed below.
+    </div>
+
     <div :class="{ 'invalid-section': documentDeliveryInvalid }">
       <v-card flat class="section-container">
-        <v-container>
-          <v-row>
-            <v-col cols="3" class="px-0">
-              <label><strong>Registered Office</strong></label>
-            </v-col>
-            <v-col cols="9" class="px-0">
-              <span class="info-text">{{getBusinessContact.email || '(Not entered)'}}</span>
-            </v-col>
-          </v-row>
-        </v-container>
-        <v-container v-if="isRoleStaff">
-          <v-row>
-            <v-col cols="3" class="px-0">
-              <label :class="{ 'error-text': documentDeliveryInvalid }"><strong>User Account</strong></label>
-            </v-col>
-            <v-col cols="9" class="px-0">
-              <v-text-field
-                v-model="optionalEmail"
-                id="optionalEmail"
-                class="text-input-field"
-                filled
-                label="Client Email Address (Optional)"
-                hint="Example: name@email.com"
-                persistent-hint
-                validate-on-blur
-                :rules="entityEmailRules"
-              />
-            </v-col>
-          </v-row>
-        </v-container>
-        <v-container v-else>
-          <v-row>
-            <v-col cols="3" class="px-0">
-              <label><strong>User Account</strong></label>
-            </v-col>
-            <v-col cols="9" class="px-0">
-              <span class="info-text">{{ getUserEmail }}</span>
-            </v-col>
-          </v-row>
-        </v-container>
+        <v-row no-gutters>
+          <v-col cols="3" class="px-0">
+            <label><strong>Registered Office</strong></label>
+          </v-col>
+          <v-col cols="9" class="px-0">
+            <span class="info-text">{{getBusinessContact.email || '(Not entered)'}}</span>
+          </v-col>
+        </v-row>
+
+        <v-row v-if="isRoleStaff" no-gutters class="mt-6">
+          <v-col cols="3" class="px-0">
+            <label :class="{ 'error-text': documentDeliveryInvalid }"><strong>User Account</strong></label>
+          </v-col>
+          <v-col cols="9" class="px-0">
+            <v-text-field
+              v-model="optionalEmail"
+              id="optionalEmail"
+              class="email-input-field mb-n2"
```

Comment:
Bottom marging (negative, too) is a special case here to make this v-text-field shorter (less unneeded whitespace at bottom).